ddmrp_adjustment/security/ir.model.access.csv

```diff
@@ -1,9 +1,9 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-ddmrp_adjustment_access_user,ddmrp.adjustment.user,model_ddmrp_adjustment,ddmrp.group_stock_buffer_maintainer,1,0,0,0
+ddmrp_adjustment_access_user,ddmrp.adjustment.user,model_ddmrp_adjustment,stock.group_stock_user,1,0,0,0
 ddmrp_adjustment_access_manager,ddmrp.adjustment.manager,model_ddmrp_adjustment,ddmrp.group_ddmrp_manager,1,1,1,1
-ddmrp_adjustment_access_sheet_user,ddmrp.adjustment.sheet.user,model_ddmrp_adjustment_sheet,ddmrp.group_stock_buffer_maintainer,1,0,0,0
```

Comment:
This line is a wizard, I don't thing it doesn't even have sense to have readonly access to a wizar. You can just remove the line.